test/stackTest.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const expect = require('chai').expect;
+const { Stack } = require('../library/stack');
+
+// creating 2 stacks for testing 
+let stack1 = new Stack(5);
+let stack2 = new Stack(3);
+
+describe('#addValuesToStack1', function() {
+// describe used to group tests - this is a groupping of tests to add/remove values from stack1
```

Comment:
add a test with stack size as 0

Reply:
add a test with stach size as letter "n"

Reply:
Added the test, please see last test.